pkg/controllermanager/controller/controllerregistration/controllerregistration_seed_control.go

```diff
@@ -277,24 +277,23 @@ func computeKindTypesForShoots(
 }
 
 type controllerRegistration struct {
-	obj          *gardencorev1beta1.ControllerRegistration
-	deployAlways bool
+	obj                  *gardencorev1beta1.ControllerRegistration
+	deployAlways         bool
```

Comment:
Please format this file :)
```
Unformatted files detected:
pkg/controllermanager/controller/controllerregistration/controllerregistration_seed_control.go
make: *** [Makefile:163: check] Error 1
```

Reply:
:( Never use Golands "replace in path" and push directly 😄